### PR TITLE
English docs update

### PR DESCRIPTION
**Pull request by stellarator**
Обновление английского раздела документации. Воссоздано из оригинальных русских файлов и присланного перевода (напрямую это вставить не получилось).

Также (по тупому) обновлена gulp-задача генерации документации.

### CODE REVIEW

File: src/menu.json [hunk 1]
```diff
@@ -224,6 +344,12 @@
                                 "uri": "maps/ru/manual/dg-ajax",
                                 "_src":"ru/manual/dg-ajax.md",
                                 "hide": false
+                            },
+                            "en": {
+                                "title": "Asynchronous requests",
```
Comment (myshov) [inline src/menu.json]: Капс у второго слова: Asynchronous Requests

File: src/doc/en/examples/markers.md [hunk 2]
```diff
@@ -1,6 +1,518 @@
...
+                        marker.openPopup();
+                    }
+                });
+            </script>
+        </body>
+    </html>
+
+### Marker with a tip
+
+The marker has two kinds tips: a regular and a static ones. Both options are considered in the following example:
```
Comment (RLRR) [inline src/doc/en/examples/markers.md]: kinds of tips

File: src/doc/en/examples/markers.md [hunk 3]
```diff
@@ -1,6 +1,518 @@
...
+                        .addTo(map)
+                        .bindLabel('I am a simple tip!');
+                });
+            </script>
+        </body>
+    </html>
+
+### Processing events of the group of markers
+
+When clicking on the marker, the map will be centered in his position:
```
Comment (RLRR) [inline src/doc/en/examples/markers.md]: his → its

File: src/doc/en/examples/vector-layers.md [hunk 4]
```diff
@@ -1,6 +1,413 @@
...
+                };
+            })();
+        }
+    })
+</script>
+
+    <!DOCTYPE html>
+    <html>
+        <head>
+            <title>Animation of the polyline renderingn</title>
```
Comment (RLRR) [inline src/doc/en/examples/vector-layers.md]: renderingn → rendering

File: src/doc/en/manual/dg-entrance.md [hunk 5]
```diff
@@ -1,11 +1,160 @@
 ## DG.Entrance
 
-Displays the entrance to the building. The class supports simple animation and display of multiple entrances.
-Entrance objects are arrows that change their scale, together with the change in scale of the map, but not below
-16 level, after which the arrows disappear from the map (until an acceptable zoom level is restored).
+Allows to display an entrance to the building. The class supports simple animation and multiple inputs display.
+Objects of entrances themselves represent arrows which change its scale together with zooming, but not lower
```
Comment (RLRR) [inline src/doc/en/manual/dg-entrance.md]: its → their

File: src/doc/en/manual/dg-label.md [hunk 6]
```diff
@@ -4,5 +4,113 @@
...
+        .setLatLng([54.9502, 82.8980]);
+        .addTo(map);
+
+#### Creation
+...
+-->
+
+#### Options
+
+When you call the <code>bindLabel</code> method, you can set the following options for the ToolTip:
```
Comment (RLRR) [inline src/doc/en/manual/dg-label.md]: ToolTip → tooltip

File: src/doc/en/manual/dg-label.md [hunk 7]
```diff
@@ -4,5 +4,113 @@
...
+            <td><code><a href="/doc/maps/en/manual/basic-types#dgpoint">Point</a></code></td>
+            <td><nobr><code>Point(12, 15)</code></nobr></td>
+            <td>The offset of a tooltip container relative to the cursor position (the position
+                of the marker in the case when used with a marker).</td>
+        </tr>
+        <tr>
+            <td><b><code>className</code></b></td>
+            <td><code>String</code></td>
+            <td><code>'dg-label'</code></td>
+            <td>The CSS class that will be assigned to the DOM element of the ToolTip.</td>
```
Comment (RLRR) [inline src/doc/en/manual/dg-label.md]: ToolTip → tooltip

File: src/doc/en/manual/dg-loading.md [hunk 8]
```diff
@@ -1,6 +1,125 @@
-## API Loading
+## Loading API
 
 {toc}
 
-<!-- TODO: translation -->
-<!-- translate whole file doc/ru/manual/dg-loading.md -->
+### Description
+
+Work with map is possible only after the code of API of maps is loaded to the browser.
```
Comment (RLRR) [inline src/doc/en/manual/dg-loading.md]: API of maps → Maps API

File: src/doc/en/manual/dg-loading.md [hunk 9]
```diff
@@ -1,6 +1,125 @@
...
+
+After the package is installed, enable it using <code>require</code>:
+
+    var DG = require('2gis-maps');
+    var map = DG.map('map', {
+        'center': [54.98, 82.89],
+        'zoom': 13
+    });
+
+Please note that when you use the npm package, there is no need to use <code>DG.then</code>m.
```
Comment (RLRR) [inline src/doc/en/manual/dg-loading.md]: Лишняя буква m в самом конце

File: src/doc/en/manual/dg-metalayers.md [hunk 10]
```diff
@@ -4,5 +4,254 @@ Allows you to create additional layers of meta information and add them to the m
...
+                or <b>transparent</b> (default):<ul><li>transparent &mdash; meta-layer receives all of the events
+                and carry them to the map</li><li>layer &mdash; meta-layer receives all of the events and stops
+                them after processing</li></ul></td>
+        </tr>
+    </tbody>
+</table>
+
+#### Events
+
+You can subscribe on the following events using
```
Comment (RLRR) [inline src/doc/en/manual/dg-metalayers.md]: subscribe on → subscribe to

File: src/doc/en/examples/base.md [hunk 11]
```diff
@@ -1,6 +1,289 @@
...
+                        });
+                    });
+                }
+            </script>
+        </body>
+    </html>
+
+### The use of connection options
+
+The following is an example of calling the cards with the following set of options: <code>pkg=basic</code>, <code>skin=light</code>.
```
Comment (RLRR) [inline src/doc/en/examples/base.md]: cards → map

File: src/doc/en/examples/markers.md [hunk 12]
```diff
@@ -1,6 +1,518 @@
...
+            map.setView([e.latlng.lat, e.latlng.lng]);
+        });
+    });
+</script>
+
+
+    <!DOCTYPE html>
+    <html>
+        <head>
+            <title>Processing of events of  markers group</title>
```
Comment (RLRR) [inline src/doc/en/examples/markers.md]: Тут двойной пробел между of и markers

File: src/doc/en/examples/popups.md [hunk 13]
```diff
@@ -1,6 +1,285 @@
-## Working with Popups
+## Work with popups
 
 {toc}
 
-<!-- TODO: translation -->
-<!-- translate whole file doc/ru/examples/popups.md -->
+### Description
+
+The following are examples of usage of pop-ups.
```
Comment (RLRR) [inline src/doc/en/examples/popups.md]: Тут перевод строки лишний

Comment (stellarator) [inline src/doc/en/examples/popups.md]: Да вроде бы не лишний (в остальных случаях форматирование такое-же)?!

Comment (RLRR) [inline src/doc/en/examples/popups.md]: В других местах вроде пишется в одну строчку, но вообще это пофиг, в markdown оно ни на что не влияет

File: src/doc/en/examples/popups.md [hunk 14]
```diff
@@ -1,6 +1,285 @@
...
+            myPopUp;
+
+        map = DG.map('map1', {
+            center: [54.98, 82.89],
+            zoom: 13
+        });
+
+        DG.popup([54.98, 82.89])
+            .setLatLng([54.98, 82.89])
+            .setContent('I'm opened by default')
```
Comment (Trufi) [inline src/doc/en/examples/popups.md]: Кавычка не экранирована

**Comment by stellarator** [comment]
Обновил, переставил на текущий мастер.
